taglib/ape/apetag.cpp

```diff
@@ -136,6 +136,13 @@ String APE::Tag::album() const
   return d->itemListMap["ALBUM"].values().toString();
 }
 
+String APE::Tag::albumArtist() const
+{
+  if(d->itemListMap["ALBUMARTIST"].isEmpty())
+    return String::null;
```

Comment:
`String::null` has been marked deprecated for some reasons. Please use an empty string like other functions of this class.
Also, please revise the related comments.